### ISSUE TITLE
Custom class and block style breaks extraction

### ISSUE THREAD

**Issue by dmgawel** [leoloso/block-metadata#2]
Most of metadata extraction rules assume that element has only one class corresponding to block name, i.e.:

https://github.com/leoloso/block-metadata/blob/c7d8dd2b0a159a7e55b67d538a6ded8cbace9088/src/Metadata.php#L174

When a custom CSS class is added via WP Block Advanced Options, or any Block Style is selected (which at the end of a day results in additional class name) it breaks parsing.

**Comment by leoloso** [comment]
Yep, I coded the logic very very simply, I didn't spend much time on it. I mainly wanted to demonstrate that the concept works... If you're willing, you can fix this issue

**Comment by leoloso** [comment]
I have not kept up with the changes in Gutenberg: when I coded it, several months ago, it was working fine. I guess they updated that block, but since I haven't not been tracking the changes, I didn't do any update. 

If you do use Gutenberg and notice that something has changed that breaks the code here, and you know what it is and you are willing to fix it, be welcome to do it
